src/main/java/com/eleks/academy/whoami/core/impl/PersistentGame.java

```diff
@@ -158,6 +164,18 @@ public void leaveGame(String player) {
 		this.currentState.peek().leavePlayer(player);
 	}
 
+	private boolean isAllPlayersSuggestCharacter() {
```

Comment:
this looks like a duplicate of the method `finished` of class SuggestingCharacters, please use old one instead of creating new one